service/organisation-service.js

```diff
@@ -46,6 +50,20 @@ async function deleteOrganisation (orgId) {
         })
 }
 
+async function rebuildRedisOrganisationsCache () {
+    debugger
```

Comment:
What is debugger?

Reply:
It is meant to pause js apps (like a breakpoint) probs shouldn't be here

Reply:
debugger is a bug!